main.go

```diff
@@ -328,29 +341,11 @@ func SaveBackupAction(c *cli.Context) error {
 		Folder:     c.String("s3-folder"),
 	}
 
-	var client = &minio.Client{}
-	var tr = http.DefaultTransport
-	var err error
-	if s3Backup {
-		client, err = setS3Service(bc, true)
-		if err != nil {
-			log.WithFields(log.Fields{
-				"s3-endpoint":    bc.Endpoint,
-				"s3-bucketName":  bc.BucketName,
-				"s3-accessKey":   bc.AccessKey,
-				"s3-region":      bc.Region,
-				"s3-endpoint-ca": bc.EndpointCA,
-				"s3-folder":      bc.Folder,
-			}).Errorf("failed to set s3 server: %s", err)
-			return fmt.Errorf("failed to set s3 server: %+v", err)
-		}
-		if bc.EndpointCA != "" {
-			tr, err = setTransportCA(tr, bc.EndpointCA)
-			if err != nil {
-				return err
-			}
-		}
-		client.SetCustomTransport(tr)
+	if backupRetries < 0 {
```

Comment:
Why are these `retries < 0` checks here? Is there some reason they would be less than zero?

Reply:
Since they were `cli.IntFlag`s they could've been negative. I was probably just too lazy to update them to `cli.UintFlag`s, fixed.